### PR TITLE
[Product Partnerships] Added SimpleWorkFlow Entity

### PR DESCRIPTION
**Pull request by rahul188**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+metaDescription: "New Relic's Amazon SimpleWorkFlow integration: what data it reports and how to enable it."
+---
+
+[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) include an integration for AWS SimpleWorkFlow, sending its metrics data to New Relic.
+
+This document explains the integration's features, how to activate it, and what data can be reported.
+
+## Features
+
+Use our integration to collect and send telemetry data to New Relic from your [SimpleWorkFlow](https://aws.amazon.com/swf/). You can monitor your services, query incoming data, and build dashboards to observe everything at a glance.
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx:16]:
```suggestion
Use our integration to collect and send telemetry data to New Relic from your [Amazon Simple Workflow Service](https://aws.amazon.com/swf/). You can monitor your services, query incoming data, and build dashboards to observe everything at a glance.
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+## Activate integration
+
+To enable this integration, see how to [connect AWS services to New Relic](/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup/).
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx:20]:
```suggestion
To enable this integration, see [how to connect AWS services to New Relic](/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup/).
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+## Find and use data
+
+To find your integration's metrics, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)** > **Metrics and events** and filter by `aws.SimpleWorkFlow`.
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx:24]:
```suggestion
To find your integration's metrics, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Metrics and events** and filter by `aws.SimpleWorkFlow`.
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+## Metric data [#metrics]
+
+This New Relic infrastructure integration collects the following Amazon SimpleWorkFlow data:
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx:28]:
```suggestion
This New Relic infrastructure integration collects the following Amazon SWF data:
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+### SimpleWorkFlow Metric data
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx:30]:
```suggestion
### SWF metric data
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+    </tbody>
+</table>
+
+All imported data has one dimension: `Domain`
+
+### Create alerts
+
+You can set up alerts to notify you if there are any changes. For example, you can set up an alert to notify relevant parties of critical or fatal errors.
+
+Learn more about creating alerts [here](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition/).
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx:188]:
```suggestion
Learn more about [creating alerts here](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition/).
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+
+Use our integration to collect and send telemetry data to New Relic from your [Amazon Simple Workflow Service](https://aws.amazon.com/swf/). You can monitor your services, query incoming data, and build dashboards to observe everything at a glance.
+
+## Activate integration
+
+To enable this integration, see [how to connect AWS services to New Relic](/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup/).
+
+## Find and use data
+
+To find your integration's metrics, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Metrics and events** and filter by `aws.SimpleWorkFlow`.
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx:24]:
```suggestion
## Get started

To enable this integration, see [how to connect AWS services to New Relic](/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup/).

Once the integration is set up, you can find your metric data by going to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Metrics and events** and filter by `aws.SimpleWorkFlow`.
```

Comment (ally-sassman) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx:24]: @rahul188 What do you think about this change?

Comment (rahul188) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-simpleworkflow-monitoring-integration.mdx:24]: @ally-sassman it seems good.

**Pull request merged**